mir_dataset_loaders/beatles.py

```diff
@@ -0,0 +1,203 @@
+"""Beatles Dataset Loader
+"""
+from collections import namedtuple
+import json
+import numpy as np
+import os
+import csv
+
+from . import BEATLES_INDEX_PATH
+from .utils import (get_save_path, get_local_path, validator, BeatData, ChordData, SectionData, KeyData,
+                    download_from_remote, RemoteFileMetadata, untar)
+
+BEATLES_DIR = 'Beatles'
+BEATLES_INDEX = json.load(open(BEATLES_INDEX_PATH, 'r'))
+BEATLES_ANNOT_REMOTE = RemoteFileMetadata(  # TODO: @rabbit this is not metadata but the annotations, do especific data type?
+    filename= os.path.join(BEATLES_DIR,'The Beatles Annotations.tar.gz'),
+    url='http://isophonics.net/files/annotations/'
+        'The%20Beatles%20Annotations.tar.gz',
+    checksum='62425c552d37c6bb655a78e4603828cc')
+
+BeatlesTrack = namedtuple(
+    'BeatlesTrack',
+    ['track_id',
+     'beats',
+     'chords',
+     'key',
+     'sections',
+     'title']
+)
+
+
+def download(data_home=None, clobber=False):
+    save_path = get_save_path(data_home)  # TODO: @rabitt have to do this so it work with data_home always, better idea?
+    save_path = get_save_path(os.path.join(save_path, BEATLES_DIR, 'annotations'))
+    download_path = download_from_remote(BEATLES_ANNOT_REMOTE, data_home=data_home, clobber=clobber)
+    untar(download_path, save_path, cleanup=True)
+    validate(data_home, silence=True)
+    print("""
+            Unfortunately the audio files of the Beatles dataset are not available for download.
+            If you have the Beatles dataset, place the contents into a folder called
+            Beatles with the following structure:
+                > Beatles/
+                    > annotations/
+                    > audio/
+            and copy the Beatles folder to {}
+        """.format(save_path))
+
+
+def validate(data_home, silence):
+    missing_files, invalid_checksums = validator(BEATLES_INDEX, data_home, silence=silence)
+    return missing_files, invalid_checksums
+
+
+def track_ids():
+    return list(BEATLES_INDEX.keys())
+
+
+def load(data_home=None, silence=False):
```

Comment:
same comment about silence - just remove the parameter everywhere. we'll fix all of the silencing issues in #17